build_package.py

```diff
@@ -15,12 +15,27 @@
 import glob
 import os
 import shutil
+import stat
 import subprocess
 import sys
 import zipfile
 
 from playwright.path_utils import get_file_dirname
 
+driver_version = "0.5.2"
+
+if not os.path.exists("driver"):
+    os.makedirs("driver")
+if not os.path.exists("playwright/driver"):
+    os.makedirs("playwright/driver")
+
+for platform in ["mac", "linux", "win32", "win32_x64"]:
+    zip_file = f"playwright-cli-{driver_version}-{platform}.zip"
+    if not os.path.exists("driver/" + zip_file):
+        url = "https://playwright.azureedge.net/builds/cli/" + zip_file
+        print("Fetching ", url)
+        subprocess.check_call(["curl", url, "-o", "driver/" + zip_file])
```

Comment:
Let's do it in memory like here? https://github.com/microsoft/playwright-python/pull/172/files#diff-56e4178dc64fcedaf003454c830bc80cR61